writers/csv_report_writer.rb

```diff
@@ -0,0 +1,16 @@
+require 'csv'
+class CsvReportWriter
```

Comment:
nice! One small nitpick is that since this lives in a `writers/` directory, it's convention to also have the class follow the same nesting structure. So in this case I'd suggest

```rb
module Writers
  class CsvReportWriter
    # ...
  end
end
```

Reply:
Looks good, updated